uitools/common/src/ImagePopupMediaItem.h

```diff
@@ -41,6 +41,8 @@ class ImagePopupMediaItem : public PopupMediaItem
 
 signals:
   void imagePopupMediaItemChanged();
+  void clickedUrl(const QUrl& url);
+  void mediaImageSourceUrl(const QUrl& sourceUrl);
```

Comment:
This is the sourceUrl for the image displayed. translates to the following images of the WebMap:
![2025-03-19_12-35-08](https://github.com/user-attachments/assets/b6c44011-a75b-45de-9c89-6df7d6a3a8e5)